source/fundamentals/logging.txt

```diff
@@ -0,0 +1,298 @@
+=======
+Logging
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+You will learn how to:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+This section gives background on the dependencies necessary to set up a
+logger and provides an example logger setup. 
+
+Background
+~~~~~~~~~~
+
+The MongoDB Java driver uses the Simple Logging Facade For Java (SLF4J).
+SLF4J allows you to specify your logging framework of choice at deployment time.
+For more information on SLF4J,
+`see the SLF4J documentation <http://www.slf4j.org/>`__.
+
+Setting up a logger is optional. When you start your application the MongoDB
+Java driver looks for the ``slf4j-api`` package in your classpath. If the driver
+can't find the ``slf4j-api`` package, the driver logs the following warning with
+``java.util.logging`` and disables all further logging:
+
+.. code-block:: none
+
+   WARNING: SLF4J not found on the classpath.  Logging is disabled for the 'org.mongodb.driver' component
+
+To set up a logger you must add the ``slf4j-api`` package, a logging framework, and a
+**binding** to your project's dependency list. A binding is a piece of code that
```

Comment:
Nit: I think the grammar of this sentence is correct, but when I first read it, I interpreted "a binding to your project's dependency list" as a single phrase, rather than "your project's dependency list" as something that all three items had to be added to.  Consider re-wording to something like:

> To set up a logger you muse add to your project's dependency list the ...

Reply:
Thanks for the note. On rereading this, I can see how it is not as clear as it could be. I took your rewording suggestion and then broke the dependencies our into an unordered list.

Fix: